kubernetes/config.aat.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  IDAM_S2S_AUTH: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+  CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+  DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
+  IDAM_URL: "https://preprod-idamapi.reform.hmcts.net:3511"
+  IDAM_OAUTH2_REDIRECT_URL: "https://rpe-bulk-scan-processor-aat.service.core-compute-aat.internal/oauth2/callback"
+  EMAIL_SMTP_SSL_TRUST: "*"
+  EMAIL_SMTP_TLS_ENABLED: "true"
+  IDAM_OAUTH2_CLIENT_ID: "bsp"
+  IDAM_S2S_AUTH_MICROSERVICE: "sscs_bulkscan"
+  LOGBACK_REQUIRE_ALERT_LEVEL: "false"
+  LOGBACK_REQUIRE_ERROR_CODE: "false"
+  ROBOTICS_EMAIL_SUBJECT: "Robotics Data"
+  ROBOTICS_EMAIL_MESSAGE: "Please find attached the robotics json file Please do not respond to this email"
```

Comment:
Is there a way we can just set the email message and subject in one place rather than copying and pasting per config file?

Reply:
Yep, we can do that